config/app-console.php

```diff
@@ -14,9 +14,10 @@
 	'controllerNamespace' => 'app\\console',
 	'bootstrap'  => ['log', 'settings'],
 	'aliases'    => [
-		'@config'=> dirname(__DIR__) . '/config',
-		'@migrations' => dirname(__DIR__) . '/database/migrations',
-		'@fixtures' => dirname(__DIR__) . '/database/fixtures',
+		'@root' => dirname(__DIR__),
```

Comment:
there is a standard alias: @webroot, pointing to the directory of the entry script. inside console -this directory equals to a root. so you don't need this alias